src/parsers/pm_io/matpower.jl

```diff
@@ -228,6 +234,14 @@ function _parse_matpower_string(data_string::String)
         gens = []
         for (i, gen_row) in enumerate(matlab_data["mpc.gen"])
             gen_data = row_to_typed_dict(gen_row, _mp_gen_columns)
+            bus_data = get(pv_bus_lookup, gen_data["gen_bus"], nothing)
+            if bus_data !== nothing
+                if bus_data["bus_type"] ∈ MP_FIX_VOLTAGE_BUSES &&
+                   bus_data["vm"] != gen_data["vg"]
+                    @info "Correcting vm in bus $(gen_data["gen_bus"]) to $(gen_data["vg"]) to match generator set-point"
+                    bus_data["vm"] = gen_data["vg"]
+                end
+            end
```

Comment:
This will work fine when all the generators that are connected to a bus have the same `vg`. If generators with different `vg` are connected to the same bus, is that a data error? Can we throw a warning for that case?

Reply:
We probably should do something to handle that case, but it could be possible to have that and only one generator available which would be valid.

Reply:
@kdheepak 109a97e

Reply:
Great. That works, although using a `Set{Int}` instead of `Dict{Int, Float64}` for tracking the corrected buses might be more appropriate.

Reply:
With `Set` I can't store the previous value.